cryptofeed/exchanges/bitmex.py

```diff
@@ -638,8 +638,7 @@ async def _order(self, msg: dict, timestamp: float):
         for data in msg['data']:
             if data.get('ordStatus'):
                 new_info = self.parse_order(data)
-                symbol = self.exchange_symbol_to_std_symbol(data['symbol'])
-                await self.callback(ORDER_INFO, feed=self.id, symbol=symbol, **new_info)
+                await self.callback(ORDER_INFO, feed=self.id, **new_info)
```

Comment:
Is `symbol` going to be included in new_info?

Reply:
Yes, however it's done at parsing stage